### PR TITLE
OPENTOK-39770: In samples repos, create a script to update Opentok SDK version reference.

### PR DESCRIPTION
**Pull request by jintgeorge**

### CODE REVIEW

**Comment by jintgeorge** [comment]
test this please

**Comment by jintgeorge** [comment]
retest this please

**Comment by jintgeorge** [comment]
@robjperez Trying another approach. Let me know what you think?

File: Basic-Video-Chat/app/build.gradle [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-        versionName "1.0"
+        targetSdkVersion targetsdkversion
+        versionCode versioncode
+        versionName versionname
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility sourcecompatibility
```
Comment (robjperez) [inline Basic-Video-Chat/app/build.gradle]: Not sure if this should be configurable. It needs to be 1_8 or it won't build.

I think I will leave JavaVersion.VERSION_1_8 fixed here

**Comment by robjperez** [comment]
Thanks for working on this @jintgeorge 
Overall the PR looks good to me, just left a comment about the java version.

File: Basic-Video-Chat/app/build.gradle (hunk 1 above)
Comment (jintgeorge) [reply Basic-Video-Chat/app/build.gradle]: I checked and it is compiling for all the projects.

**Pull request merged**